stairctl/src/main/java/bio/terra/stairctl/commands/Output.java

```diff
@@ -0,0 +1,222 @@
+package bio.terra.stairctl.commands;
+
+import bio.terra.stairctl.ConnectParams;
+import bio.terra.stairway.Control;
+import bio.terra.stairway.Control.KeyValue;
+import bio.terra.stairway.Control.LogEntry;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
I'm terrible at envisioning format outputs in my head, if you don't mind posting a couple examples of the  output of chattier commands, that would be helpful.

Reply:
Would it be OK if I put that in the STAIRCTL.md?

Reply:
STAIRCTL.md would be perfect